README.md

```diff
@@ -1,11 +1,11 @@
-# Bolt for Python
+# Bolt ![Bolt logo](docs/assets/bolt-logo.svg) for Python (beta)
 
 [![Python Version][python-version]][pypi-url]
 [![pypi package][pypi-image]][pypi-url]
 [![Build Status][travis-image]][travis-url]
 [![Codecov][codecov-image]][codecov-url]
 
-A Python framework to build Slack apps in a flash with the latest platform features. Check the [document](https://slack.dev/bolt-python/) and [examples](https://github.com/slackapi/bolt-python/tree/main/examples) to know how to use this framework.
+A Python framework to build Slack apps in a flash with the latest platform features. Read the [getting started guide](https://slack.dev/bolt-python/tutorial/getting-started) and look at our [code samples](https://github.com/slackapi/bolt-python/tree/main/samples) to learn how to build apps using Bolt.
```

Comment:
We updated the `samples/` directory to `examples/` to create some consistency across all our SDKs and frameworks: https://github.com/slackapi/bolt-python/tree/main/examples

We can call it "code samples" or "code examples" but using the word "examples" may help hint what folder to look for:

```suggestion
A Python framework to build Slack apps in a flash with the latest platform features. Read the [getting started guide](https://slack.dev/bolt-python/tutorial/getting-started) and look at our [code examples](https://github.com/slackapi/bolt-python/tree/main/examples) to learn how to build apps using Bolt.
```